src/suites/cts/command_buffer/copies.spec.ts

```diff
@@ -109,3 +109,67 @@ g.test('b2t2t2b', async t => {
 
   t.expectContents(dst, data);
 });
+
+g.test('imageBitmapToTexture', async t => {
+  const img = document.createElement('img');
+  img.src = '../../assets/img/test.png';
+  await img.decode();
+
+  const imageBitmap = await createImageBitmap(img, 0, 0, img.width, img.height);
+  const dst = t.device.createTexture({
+    size: {
+      width: img.width,
+      height: img.height,
+      depth: 1,
+    },
+    format: 'rgba8uint',
+    usage: GPUTextureUsage.COPY_DST | GPUTextureUsage.COPY_SRC,
+  });
+
+  // TODO(shaobo.yan@intel.com) : remove as any after @webgpu/types updating
+  (t.device.defaultQueue as any).copyImageBitmapToTexture(
+    {
+      imageBitmap,
+      origin: {
+        x: 0,
+        y: 0,
+      },
+    },
+    {
+      texture: dst,
+    },
+    {
+      width: imageBitmap.width,
+      height: imageBitmap.height,
+      depth: 1,
+    }
+  );
+
+  const imageCanvas = document.createElement('canvas');
+  imageCanvas.width = imageBitmap.width;
+  imageCanvas.height = imageBitmap.height;
+
+  const imageCanvasContext = imageCanvas.getContext('2d');
+  t.expect(imageCanvasContext != null);
+  if (!imageCanvasContext) {
+    return;
+  }
+
+  imageCanvasContext.drawImage(imageBitmap, 0, 0, imageBitmap.width, imageBitmap.height);
+  const data = imageCanvasContext.getImageData(0, 0, imageBitmap.width, imageBitmap.height).data;
+
+  const testBuffer = t.device.createBuffer({
+    size: imageBitmap.width * imageBitmap.height * 4,
+    usage: GPUBufferUsage.COPY_SRC | GPUBufferUsage.COPY_DST,
+  });
+
+  const encoder = t.device.createCommandEncoder();
+  encoder.copyTextureToBuffer(
+    { texture: dst, mipLevel: 0, origin: { x: 0, y: 0, z: 0 } },
+    { buffer: testBuffer, rowPitch: imageBitmap.width * 4, imageHeight: imageBitmap.height },
```

Comment:
We should explicitly compute the rowPitch with the 256-byte alignment. If this works, then I think we're getting lucky the the image width is already a multiple of 256.

The size of `testBuffer` would become `rowPitch * imageBitmap.height`

Reply:
yeah, source image is 512x512.

Reply:
Yes, it happens to be a multiple of 256 so I directly use it. But yes, I can add a logic to compute the rowPitch to make it more general

Reply:
Added a helper funtion for rowPtich